__tests__/validateCarNames.test.js

```diff
@@ -14,7 +20,7 @@ describe("CarNames 유효성 테스트", () => {
       input: "Niya,Niya",
```

Comment:
전반적으로 변수화 하여 표현하면 더 명확할 것 같아요.
const name1 = "Niya";
const name2 = "HoyyChoi";

```suggestion
      input: `${Niya},${Niya}`,
```